ProcessesApi/V1/Gateways/SoleToJointGateway.cs

```diff
@@ -137,11 +137,11 @@ private bool BR19(Person proposedTenant)
         }
 
         /// <summary>
-        /// Passes if the proposed tenant does not have any active tenures that are not non-secure
+        /// Passes if the proposed tenant does not have any active tenures (other than the selected tenure) that are not non-secure
         /// </summary>
-        private bool BR9(Person proposedTenant)
+        private bool BR9(Person proposedTenant, Guid tenureId)
         {
-            return !proposedTenant.Tenures.Any(x => x.IsActive && x.Type != TenureTypes.NonSecure.Code);
+            return !proposedTenant.Tenures.Any(x => x.IsActive && x.Type != TenureTypes.NonSecure.Code && x.Id != tenureId);
```

Comment:
Why is the non-secure check also here?

Reply:
BR9 is based on "Is the proposed tenant a tenure holder or household member of an existing tenure within the London Borough of Hackney?" - but the check does not limit that to only tenures, which are not non secure.

Reply:
In the spreadsheet, it says this: "Check tenure records - active dwelling either as a household member or a named tenure holder in any tenure type which isnt non secure (issue is that someone could have a decant)", so that's what I went with. Not sure if this has changed because there was a comment on this saying to discuss with Gareth and Michael.